hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/DeltaSync.java

```diff
@@ -495,6 +498,8 @@ public Pair<SchemaProvider, Pair<String, JavaRDD<HoodieRecord>>> readFromSource(
           .setPartitionMetafileUseBaseFormat(props.getBoolean(HoodieTableConfig.PARTITION_METAFILE_USE_BASE_FORMAT.key(),
               HoodieTableConfig.PARTITION_METAFILE_USE_BASE_FORMAT.defaultValue()))
           .setShouldDropPartitionColumns(isDropPartitionColumns())
+          .setHiveStylePartitioningEnable(props.getBoolean(HIVE_STYLE_PARTITIONING_ENABLE.key(),
```

Comment:
Does `urlEncodePartitioningEnabled` write to hoodie.properties or not? If not, I think we can add it as well.

Reply:
Nope, not written to `hoodie.properties`.

```properties
#Properties saved on 2023-05-18T07:48:15.181Z
#Thu May 18 15:48:15 SGT 2023
hoodie.table.precombine.field=timestamp
hoodie.datasource.write.drop.partition.columns=false
hoodie.table.partition.fields=partition_path
hoodie.table.type=COPY_ON_WRITE
hoodie.archivelog.folder=archived
hoodie.table.cdc.enabled=false
hoodie.populate.meta.fields=true
hoodie.partition.metafile.use.base.format=false
hoodie.table.version=5
hoodie.timeline.layout.version=1
hoodie.table.base.file.format=PARQUET
hoodie.table.recordkey.fields=_row_key
hoodie.table.name=hoodie_trips
hoodie.table.keygenerator.class=org.apache.hudi.utilities.deltastreamer.TestHoodieDeltaStreamer$TestGenerator
hoodie.datasource.write.hive_style_partitioning=true
hoodie.table.checksum=224269229
```

Reply:
@boneanxs Done!